x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -166,7 +167,40 @@ export async function getKibanaAssets(
   return result;
 }
 
-async function installKibanaSavedObjects({
+const isImportConflictError = (e: SavedObjectsImportFailure) => e?.error?.type === 'conflict';
+/**
+ * retry saved object import if only conflict errors are encountered
+ */
+async function retryImportOnConflictError(
+  importCall: () => ReturnType<SavedObjectsImporterContract['import']>,
+  {
+    logger,
+    maxAttempts = 5,
+    _attempt = 0,
+  }: { logger?: Logger; _attempt?: number; maxAttempts?: number } = {}
+): ReturnType<SavedObjectsImporterContract['import']> {
+  const result = await importCall();
+  const errors = result.errors ?? [];
+
+  if (_attempt < maxAttempts && errors.length && errors.every(isImportConflictError)) {
+    const retryCount = _attempt + 1;
+    const retryDelaySec = Math.min(Math.pow(2, retryCount), 64); // @dover's patented retry, 2s, 4s, 8s, 16s, 32s, 64s, 64s, 64s ...
+
+    logger?.warn(
```

Comment:
Not sure if warn is overkill here?

Reply:
++ let's downgraded to debug since this is basically expected behavior and isn't actionable for the user